wsbroadcastserver/clientmanager.go

```diff
@@ -109,7 +109,7 @@ func (cm *ClientManager) removeAll() {
 }
 
 func (cm *ClientManager) removeClientImpl(clientConnection *ClientConnection) {
-	clientConnection.StopAndWait()
+	clientConnection.StopOnly()
```

Comment:
Careful here: this doesn't call the client connection's existing StopAndWait code. We should either remove that or maybe rename it to StopOnly and have it wrap that

Reply:
:+1: